src/optimistic-oracle.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <include src="head.html"></include>
+  <body>
+    <div class="wrapper">
+      <include src="header.html"></include>
+
+      <section class="products-hero">
+        <span class="products-uma-blurb">
+          <img
+            class="products-sm-logo"
+            src="assets/images/powered_by_uma.png"
+            alt="UMA_logo"
+        /></span>
+        <div class="shell">
+          <div class="hero__inner">
+            <h1>Optimistic Oracle</h1>
+
+            <h4>
+              The Optimistic Oracle is used to retrieve prices or other data and
+              put them on-chain. It has an inbuilt resolution system in case of
```

Comment:
i think there should be a mention of how this contrasts with dvm, ie the OO is faster and cheaper, something to that effect, @Tommy1231232